lib/bundler.js

```diff
@@ -55,6 +55,18 @@ function bundle(options) {
     var sourcePacksContent = Fs.readFileSync(sourcePacksFile).toString();
     // Write the composed content to the temp packages file
     Fs.writeFileSync(tempPacksFile, sourcePacksContent);
+    //copy thec custom packages folder into the tempPacksDir if it exists
+    var sourceCustomPackagesFolder = Path.resolve(sourceDir, "packages/")
+    if(Fs.existsSync(sourceCustomPackagesFolder)){
+      var tempCustomPackagesFolder = Path.resolve(tempDir, "packages/")
```

Comment:
- Use `tempCustomPackagesDir` instead of `tempCustomPackagesFolder`.
- You don't need the trailing `/` in `packages`.